src/components/books/Book.module.scss

```diff
@@ -14,13 +14,32 @@
 
   &__title {
     line-height: 1.5;
+    margin-top: 0.188rem;
+    font-size: 1.375rem;
+    letter-spacing: -0.2px;
+    font-weight: 700;
+    color: #121212;
+  }
+
+  &__categories {
+    font-weight: 700;
+    color: #121212;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 0.875rem;
+    opacity: 0.5;
```

Comment:
- [ ] As mentioned by the previous reviewer, It is required to produce the exact design of the sample template. This includes the layout, color, font-family among others. At this moment, your font-family is different from the one used on the sample template and the color of some elements. Kindly look into the two designs and work your design alongside the sample template. All the best.

### Project Requirement.
![project-rqts](https://user-images.githubusercontent.com/26597432/215270719-24ab68a5-8d70-42e3-95ec-07472dcf9e06.png)

### Your Design
![your-page](https://user-images.githubusercontent.com/26597432/215270740-fb369485-6310-4a24-a8c2-12cd6a542237.png)

### Sample Template
![temp-page](https://user-images.githubusercontent.com/26597432/215270751-76a1b072-9c31-4162-9f1c-fa74cee31e68.png)

Reply:
@zieeco thanks for your recommendation.

Reply:
Fixed.